Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -1009,7 +1041,7 @@ script:
       name: retries
       required: true
     description: Get PAPI property activation status until it is active.
-    name: akamai-get-papi-property-activation-status-command
+    name: akamai-get-papi-property-activation-status
```

Comment:
This is a correct fix
but that is not possible
because it breaks backward compatibility
therefore, it should be reverted

Reply:
This has been reverted.